master/.goreleaser.yml

```diff
@@ -12,7 +12,7 @@ builds:
     ldflags:
       - -X github.com/determined-ai/determined/master/version.Version={{.Env.VERSION}}
       - -X github.com/determined-ai/determined/master/internal/config.DefaultSegmentMasterKey={{.Env.DET_SEGMENT_MASTER_KEY}}
-      - -X github.com/determined-ai/determined/master/internal/config.DefaultSegmentWebUIKey={{.Env.DET_SEGMENT_WEBUI_KEY}}
+      - -X github.com/determined-ai/determined/master/internal/config.DefaultSegmentWebKey={{.Env.DET_SEGMENT_WEBUI_KEY}}
```

Comment:
using different environment variable: problematic?

i.e. would the cluster get a new segment key??

Reply:
@dzhu will know this best, but the config name change should be fine. The renaming of the env key name will require coordination on segment.io

Reply:
yeah I was thinking we would probably leave this unchanged. 

but for initial draft PR i wanted to do a complete purge for illustration purposes, then we can walk things back to wherever makes sense.

Reply:
I think the CircleCI configuration (https://app.circleci.com/settings/project/github/determined-ai/determined/environment-variables and some of the contexts under https://app.circleci.com/settings/organization/github/determined-ai/contexts) would have to change, not Segment.

Also, I would guess that this is indeed all fine to change; technically the environment variable and CLI option are user-facing, but I highly doubt anyone is using them. (I suppose you should confirm with more people than just me, though.)

Reply:
okay thanks, maybe @shiyuann  can confirm?